### ISSUE TITLE
Recent table changes break inline code parsing in tables.

### ISSUE THREAD

**Issue by facelessuser** [Python-Markdown/markdown#444]
Recent changes to tables cause the trailing character after an inline code block to get swallowed up into the ether never to be seen again (notice the trailing `.` and `s` disappear).  This only occurs inside tables.

``` python
>>> import markdown
>>> src = '''
... | column 1 | column 2 |
... | -------- | -------- |
... | `broken`.| `code`s  |
... 
... '''
>>> markdown.markdown(src, extensions=["markdown.extensions.tables"])
u'<table>\n<thead>\n<tr>\n<th>column 1</th>\n<th>column 2</th>\n</tr>\n</thead>\n<tbody>\n<tr>\n<td><code>broken</code></td>\n<td><code>code</code></td>\n</tr>\n</tbody>\n</table>'
```

**Comment by facelessuser** [comment]
This looks to be a duplicate of #440.  Looks like I will have to downgrade until the fix is released.

**Comment by waylan** [comment]
Sorry, for some reason I thought that I had made a new release since the fix to #440 was merged.  In any event, I just pushed up a new release (2.6.5) with the fix.

**Comment by facelessuser** [comment]
Thanks!
